### PR TITLE
feat(complex-rubric): add validate logic PD-3260

### PR DESCRIPTION
**Pull request by arimieandreea**

### CODE REVIEW

**Comment by arimieandreea** [comment]
https://illuminate.atlassian.net/browse/PD-3260

File: packages/complex-rubric/controller/src/index.js [hunk 1]
```diff
@@ -88,3 +89,119 @@ export const createCorrectResponseSession = (question, env) => {
...
+  if(Object.keys(scorePointsErrors).length > 0){
+    errors.scorePointsErrors = scorePointsErrors;
+  }
+
+  return errors;
+};
+
+
+export const validate = (model = {}, config = {}) => {
+  console.log(model);
```
Comment (andreeapescar) [inline packages/complex-rubric/controller/src/index.js:191]: Remove log.

File: packages/complex-rubric/controller/src/index.js [hunk 2]
```diff
@@ -88,3 +89,119 @@ export const createCorrectResponseSession = (question, env) => {
...
+  });
+
+  if(Object.keys(pointsDescriptorsErrors).length > 0){
+    errors.pointsDescriptorsErrors = pointsDescriptorsErrors;
+  }
+
+  return errors;
+};
+
+// this should always be updated when validate function of multi-trait-rubric is changed (as for ebsr)
```
Comment (andreeapescar) [inline packages/complex-rubric/controller/src/index.js:119]: “IMPORTANT! This function is a duplicate of multi-trait-rubric/controller/validate function! If you make any changes to this function, please make sure you also update multi-trait-rubric/controller/validate function!“.


Add this comment (of course, with proper "multi-trait-rubric/controller/validate function") in rubric, multi-trait-rubric and both validation functions in complex-rubric.

File: packages/complex-rubric/controller/src/index.js [hunk 3]
```diff
@@ -88,3 +89,119 @@ export const createCorrectResponseSession = (question, env) => {
     }
   });
 };
+
+// this should always be updated when validate function of rubric is changed (as for ebsr)
+const validateSimpleRubric = (model) => {
```
Comment (andreeapescar) [inline packages/complex-rubric/controller/src/index.js:94]: “IMPORTANT! This function is a duplicate of rubric/controller/validate function! If you make any changes to this function, please make sure you also update rubric/controller/validate function!“.

**Review by andreeapescar — CHANGES_REQUESTED** [review]

**Pull request merged**